repose-aggregator/components/filters/api-validator-filter/src/main/java/org/openrepose/filters/apivalidator/ValidatorConfigurator.java

```diff
@@ -61,7 +60,9 @@ public ValidatorInfo getDefaultValidator() {
     }
 
     public void processConfiguration(ValidatorConfiguration validatorConfiguration, String configRoot, String wadlUri) {
-        defaultvalidator = null;
+        logDeprecationWarnings(validatorConfiguration);
```

Comment:
I really like this being pulled out to it's own method so it can be easily removed later.